ci-operator/step-registry/baremetal/lab/ipi/install/baremetal-lab-ipi-install-commands.sh

```diff
@@ -162,6 +162,11 @@ do
   cp "${item}" "${INSTALL_DIR}/manifests/${manifest##manifest_}"
 done < <( find "${SHARED_DIR}" \( -name "manifest_*.yml" -o -name "manifest_*.yaml" \) -print0)
 
+## HW RAID configuration
```

Comment:
Do we have to uncomment to reconcile the env var setting in `HW_RAID` in baremetal-lab-ipi-install-ref.yaml?